src/components/SecretsTable/SecretsTable.js

```diff
@@ -90,6 +90,9 @@ const SecretsTable = props => {
           selectedRows
         }) => (
           <TableContainer>
+            <div className="header">
+              <h4 className="bx--data-table-header__title">Secrets</h4>
```

Comment:
the TableContainer component has a `title` prop that should be used instead

`<TableContainer title="Secrets">`